config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -768,7 +768,7 @@ periodics:
     testgrid-tab-name: aks-engine-windows-containerd-azure-file-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Run Azure File e2e test with Azure File in a Windows cluster configured with CSI proxy and containerd.
-- interval: 12h
+- interval: 24h
```

Comment:
Any reason to not keep the containerd-nightly run at 12 hours?

Reply:
The thinking was: the main focus for this test is to make sure the containerd/hcsshim don't have major breaking changes. Since changes going into containerd/hcsshim are not as frequent as k/k I didn't think we need to run it twice a day since we have coverage of the k/k changes via the 3hr containerd build.

Reply:
Sounds good.